CHANGELOG.md

```diff
@@ -18,4 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
-## [1.0.0] - 2021.12.12
\ No newline at end of file
+## [1.0.0] - 2021.12.12
+
+## [1.1] - TBD
```

Comment:
Refer to my comment on your other PR: [Comment](https://github.com/mietechnologies/MIE-MCServer/pull/18#discussion_r773504495)